<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartMirror</class>
 <widget class="QMainWindow" name="SmartMirror">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartMirror</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="weather">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>170</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Weather</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sports">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>170</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Sports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stocks">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>400</y>
      <width>170</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stocks</string>
    </property>
   </widget>
   <widget class="QPushButton" name="news">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>400</y>
      <width>170</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>News</string>
    </property>
   </widget>
   <widget class="QPushButton" name="memes">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>400</y>
      <width>170</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Memes</string>
    </property>
   </widget>
   <widget class="QLabel" name="mainscreen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>852</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="meme1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>852</width>
      <height>360</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="memeName">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>852</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="header">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>429</y>
      <width>851</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stock">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>120</y>
      <width>301</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lastGame">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>250</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="nextGame">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>341</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="teamHeader">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="city">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="tempLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="feelsLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="feelsValue">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>21</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="leagueBox">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>NBA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NHL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>NFL</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="teamBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>190</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="weatherDesc">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="stockInput">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="enterStock">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>100</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stockHeader">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="forecastTitle">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dayOne">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dayTwo">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dayThree">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>260</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="stockEnter">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="stockTicker">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stockOpen">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>180</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stockHigh">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>210</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="stockLow">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="weatherIcon">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>181</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>852</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
